rdr_service/resource/generators/participant.py

```diff
@@ -350,11 +350,18 @@ def _prep_participant(self, p_id, ro_session):
         # credentials but are not correctly flagged in the participant table
         test_participant = p.isGhostId == 1 or p.isTestParticipant == 1 or (hpo and hpo.name == TEST_HPO_NAME)
 
+        # TODO: Workaround for PDR-364 is to pull cohort value from participant_summary. LIMITED USE CASE ONLY
+        cohort = ConsentCohortEnum.UNSET if not p or not p.participantSummary \
```

Comment:
This may be fine as well.  I was thinking of adding this down around line 430 where we have other dependencies on ParticipantSummary for things that we ultimately thought might go in the proposed `participant_profile `table.

I'll leave it up to you if you want to do it here or there